besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -124,12 +133,67 @@ public void run() {
       final Path dataDirectoryPath =
           Paths.get(
               TrieLogSubCommand.parentCommand.parentCommand.dataDir().toAbsolutePath().toString());
+
+      LOG.info("Estimating trie logs size before pruning...");
+      long sizeBefore = estimatedSizeOfTrieLogs();
+      LOG.info("Estimated trie logs size before pruning: {}", formatOutputSize(sizeBefore));
+
       final TrieLogHelper trieLogHelper = new TrieLogHelper();
-      trieLogHelper.prune(
-          context.config(),
-          context.rootWorldStateStorage(),
-          context.blockchain(),
-          dataDirectoryPath);
+      boolean success =
+          trieLogHelper.prune(
+              context.config(),
+              context.rootWorldStateStorage(),
+              context.blockchain(),
+              dataDirectoryPath);
+
+      if (success) {
+        LOG.info("Finished pruning. Re-estimating trie logs size...");
```

Comment:
Similar to above comment unless this takes any significant time I think users only care about the next line with the estimate of the trie log size.

Reply:
This second one was quicker than the first in my testing, but even 10 seconds is a long time for users to be uncertain if a command is working I think.